### ISSUE TITLE
No explanation, no examples

### ISSUE THREAD

**Issue by salgiza** [MicrosoftDocs/dynamics365smb-devitpro-pb#516]
Ok, so this Data Type  exists and has methods, that is as far as the help goes. There is an example in another section of the docs on how to use this Data Type (not linked here, of course) that shows that this class has to be used when publishing procedures through OData.

What does this Data Type do? Not explained.
What does any of the methods do?  Not explained.
How should I use any of it? Not explained.

Soooo... I guess that the only way to be able to publish procedures through OData in Business Central is by finding out who was the developer in charge of implementing this feature and ask him/her by email. Or something like that.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 25bff681-daab-1f20-5017-fb68c9d1bb1c
* Version Independent ID: a877c6c8-6885-632a-e136-a4acd8b6fa14
* Content: [WebServiceActionContext Data Type - Business Central](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/webserviceactioncontext/webserviceactioncontext-data-type)
* Content Source: [dev-itpro/developer/methods-auto/webserviceactioncontext/webserviceactioncontext-data-type.md](https://github.com/MicrosoftDocs/dynamics365smb-devitpro-pb/blob/live/dev-itpro/developer/methods-auto/webserviceactioncontext/webserviceactioncontext-data-type.md)
* Service: **dynamics365-business-central**
* GitHub Login: @solsen
* Microsoft Alias: **solsen**

**Comment by SusanneWindfeldPedersen** [comment]
Hello, I agree these topics do not provide much context. We'll work on improving them, but in the meantime please find this [topic](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-creating-and-interacting-with-odatav4-bound-action) for an example of how to use the method. Hope this helps a bit.... Thanks.

**Comment by salgiza** [comment]
I understand from the "closed" tag that there is no intention to fix the docs?

The link you provided is the one where I discovered that this Data Type and methods existed, but without any information I've been unable to use them (the example didn't work, but how am I going to debug it if I'm just copying-pasting code without any idea as to what really does?).

I finally skipped using odata procedures and went with traditional SOAP web services instead. This is what every other partner I know has finally ended up doing, but it is really sad: why spend time implementing this features, and then let them die unused because no-one bothered to document how to use them?
